### PR TITLE
feat: [GIT-619]: pass token and RbacMenuItem to child app

### PR DESCRIPTION
**Pull request by vkbansal**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Review by abhinavrastogi-harness — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Review by abhinavrastogi-harness — APPROVED** [review]

**Review by reetika04 — APPROVED** [review]

**Comment by vkbansal** [comment]
retrigger jest

File: src/modules/75-cd/components/DashboardList/DashboardList.tsx [hunk 1]
```diff
@@ -7,9 +7,9 @@
 
 import React, { useCallback, useContext, useEffect, useState } from 'react'
 import { Color, ExpandingSearchInput, Layout, Text, PageError, TableV2 } from '@wings-software/uicore'
+import type { TableProps } from '@harness/uicore'
```
Comment (knagurski) [inline src/modules/75-cd/components/DashboardList/DashboardList.tsx:10]: Any reason to not combine these with the `@wings-software/uicore` imports?

File: src/modules/85-cv/components/CVSetupSourcesView/SetupSourceMappingList/SetupSourceMappingList.tsx [hunk 2]
```diff
@@ -32,7 +33,7 @@ interface TableFilterForSetupSourceMapping<T> extends Omit<TableFilterProps, 'on
   onFilterForMoreThan100Items?: (filterString: string) => void
 }
 export interface SetupSourceMappingListProps<T extends Record<string, unknown>> {
-  tableProps: TableProps<T>
+  tableProps: Omit<TableProps<T>, 'bpTableProps'> & { bpTableProps?: TableProps<T>['bpTableProps'] }
```
Comment (knagurski) [inline src/modules/85-cv/components/CVSetupSourcesView/SetupSourceMappingList/SetupSourceMappingList.tsx:36]: DOn't ya just love how some TypeScript is just like poetry :-p

**Review by harshilbits — APPROVED** [review]

File: src/modules/75-cd/components/DashboardList/DashboardList.tsx (hunk 1 above)
Comment (knagurski) [reply src/modules/75-cd/components/DashboardList/DashboardList.tsx:10]: Thinking is WAAAAYYYYY overrated!

**Pull request merged**